RapidRiverSoftware/rapidriversoftware.github.io
# Feature | Blue headers

### This PR adds:
1. Updated header images
2. Updates header styles
3. Removal or outdated styles
4. Minor UI fixes

### To Test:
1. Follow the instructions in the README to start the project
2. Check that headers have all changed
3. Check that footer has changed too
4. Click on the phone number and address to test "callable" and "google mappable", respectively
